tooling/src/publish.js

```diff
@@ -28,46 +28,42 @@ function versionAndPublish() {
     process.exit(1);
   }
   const contactCenterPath = './packages/contact-center';
-  const dependencies = ['@webex/cc-store'];
 
   try {
-    const ccFolder = fs
+    const workspaceData = fs
       .readdirSync(contactCenterPath, {withFileTypes: true})
-      .filter((dirent) => {
-        return dirent.isDirectory();
-      })
+      .filter((dirent) => dirent.isDirectory())
       .map((dirent) => {
-        try {
-          const packageJsonPath = path.join(contactCenterPath, dirent.name, 'package.json');
-          const packageData = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
-
-          console.log(`Removing stable version from package.json for ${dirent.name}`);
-          removeStableVersion(packageJsonPath, packageData);
-          return packageData.name;
-        } catch (error) {
-          throw new Error(`Error reading package.json in ${dirent.name}`, error);
+        const packageJsonPath = path.join(contactCenterPath, dirent.name, 'package.json');
+        if (!fs.existsSync(packageJsonPath)) {
+          throw new Error(`package.json not found in ${dirent.name}`);
         }
-      });
-    // Separate dependency workspaces and other workspaces
-    const dependencyWorkspaces = ccFolder.filter((fileName) => dependencies.includes(fileName));
+        const packageData = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
 
-    const otherWorkspaces = ccFolder.filter((fileName) => !dependencies.includes(fileName));
-
-    const publishWorkspace = (workspaceName) => {
-      console.log(`Publishing new version for ${workspaceName}: ${newVersion}`);
+        console.log(`Removing stable version from package.json for ${dirent.name}`);
```

Comment:
I see console logs in this file, we would remove those before merge right

Reply:
These logs for debugging the publish step. They will only appear in the circleCI